frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -42,6 +39,8 @@ action.GitMachete.PullCurrentBranchFastForwardOnlyBackgroundable.operation-name=
 
 action.GitMachete.BaseFastForwardParentToMatchBranchAction.description-action-name=Fast forward
 action.GitMachete.BaseFastForwardParentToMatchBranchAction.description=Fast forward branch ''{0}'' to match ''{1}''
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.notification.title.ff-fail=Fast forward of branch ''{0}'' to match ''{1}'' failed
```

Comment:
Nit: hmm so `'branch'` in failure notification and `<b>branch</b>` in success notifications?

Reply:
Please have a look at the samples above (screenshots).

Note that success notifications show up in a different place than error notifications. 
The msg text of the second one is bold already so we need indicate it differently (with `'`)